crates/document_parse/Cargo.toml

```diff
@@ -10,7 +10,7 @@ version.workspace = true
 
 [dependencies]
 docx-rs = { git = "https://github.com/spiceai/docx-rs", rev="2d86a60b58afb02b157f96b42a92626417e47f71"}
-pdf-extract = { git = "https://github.com/spiceai/pdf-extract", rev = "3ca5e2c904cdf5897f3d671e3b2c4b62b1612b0c" }
+pdf-extract = { git = "https://github.com/kczimm/pdf-extract.git", rev = "e2a07e5fe85de13484342c2ea1e0cd654a8cfce3" }
```

Comment:
Let's keep this pointed at the `spiceai` fork.

Reply:
You should make a PR into the fork.

Reply:
I saw we used a branch of spiceai/pdf-extract fork. I didn't know if our policy is to operate out of branches on our forks. If so, I need permission to create a branch there otherwise I can create a PR to merge into a main.